env0/resource_template_test.go

```diff
@@ -697,44 +713,6 @@ func TestUnitTemplateResource(t *testing.T) {
 		})
 	}
 
-	t.Run("Should not trigger terraform changes when gitlab_project_id is provided", func(t *testing.T) {
```

Comment:
This test is incorrect.
It should trigger changes.

See: https://developer.env0.com/docs/api/c667ff5b5fecb-update-template